rules/windows/defense_evasion_sip_provider_mod.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2021/01/20"
+maturity = "production"
+updated_date = "2021/01/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies modifications to the registered Subject Interface Package (SIP) providers. SIP providers are used by the
+Windows cryptographic system to validate file signatures on the system. This may be an attempt to bypass signature
+validation checks or inject code into critical processes.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "SIP Provider Modification"
```

Comment:
maybe u can add this as a reference https://github.com/mattifestation/PoCSubjectInterfacePackage

Reply:
added, thanks!